dev/com.ibm.websphere.appserver.features/visibility/public/jwtSso-1.0/com.ibm.websphere.appserver.jwtSso-1.0.feature

```diff
@@ -5,7 +5,7 @@ singleton=true
 IBM-ShortName: jwtSso-1.0
 # we don't need servlet 4.0, but we specify it to suppress
 # a CWWKF0001E from jwt's use of servlet 3.0
-Subsystem-Name: JwtSso
+Subsystem-Name: Jwt SSO 1.0
```

Comment:
my vote is still for "JwtSso -> JSON Web Token Single Sing-On 1.0" over "Jwt SSO 1.0" or at least "JWT SSO 1.0"